src/components/v2/values/chartValuesDiff/ChartValuesView.reducer.ts

```diff
@@ -121,6 +121,8 @@ export const chartValuesReducer = (state: ChartValuesViewState, action: ChartVal
             return { ...state, forceDeleteData: action.payload }
         case ChartValuesViewActionTypes.errorResponseCode:
             return { ...state, errorResponseCode: action.payload }
+        case ChartValuesViewActionTypes.invalidValueName:
```

Comment:
Can we add a default state for `invalidValueName` in initState?

Reply:
done

Reply:
Also, invalidValueNameMessage action seems to be missing.